signac/contrib/job.py

```diff
@@ -644,7 +647,7 @@ def remove(self):
                     if not error.errno == errno.ENOENT:
                         raise error
                 self._document = None
-            self._data = None
+            self._stores = None
```

Comment:
This is a bug left in from long ago. The `_data` attribute doesn't exist, it's called `_stores`.